OMCompiler/Compiler/FrontEnd/SCode.mo

```diff
@@ -5868,7 +5868,7 @@ public function isEmptyMod
   output Boolean isEmpty;
 algorithm
   isEmpty := match mod
-    case Mod.NOMOD() then true;
```

Comment:
@sjoelund Är det här gammal MetaModelica stil för att skriva det som jag ändrade det hela till?

Reply:
NOMOD() är gamla, Mod.NOMOD() är nyare. I min PR ändrade jag en del av de nya till de gamla för att få mina search-and-replace att fungera lättare när det var blandat.

Reply:
Den nya stilen är lite problematisk. Abstrakta typer i Julia kan ju inte riktigt ha fält på samma sätt som i MetaModelica (Objektmodellen (Rätt ord?) Är annorlunda). Jag tror inte heller att jag har typinformation att skilja på dessa fall direkt i Absyn